src/tests/classic/run.rs

```diff
@@ -304,6 +304,21 @@ pub struct RandomClvmNumber {
     pub intended_value: Number,
 }
 
+#[test]
+fn test_classic_mod_form() {
```

Comment:
classic mod form means that you're allowed to nest it?
is this something we should discourage? we could issue a deprecation warning for instance.

Reply:
It's not my favorite thing.  From a language perspective, given other features, I think it's surprising (for example, it isn't a closure it's a blank slate) unless you're really into chialisp.  I put off including it in modern until it was needed.  That having been said, this particular test is just adding a specific test of this feature in class; the one in tests/compiler/compiler.rs tests the modern variant.  Balancing things people wanted and already did, along with the fair criticism that it isn't currently highlighted in the language server and would need to be ported using a macro, I decided to add it at this point.  Like C++ we can add other features alongside.

Reply:
I'll think about a deprecation warning.  Classically "run" uses stdout to report errors and runs in the hidden context of python at load time, so i've tried to be conservative and mimic the existing tools, but probably to the detriment of usability in a lot of cases.